readme/Configuration.scalatex

```diff
@@ -206,7 +206,7 @@
        */
 
   @sect{newlines.alwaysBeforeTopLevelStatements}
-    Default: default.newlines.alwaysBeforeTopLevelStatements
+    Default: @b(default.newlines.alwaysBeforeTopLevelStatements)
```

Comment:
this is the most important part of the diff. The missing `@` results in:

![image](https://cloud.githubusercontent.com/assets/344610/23708219/1f8f5bba-040d-11e7-9822-d8b2a10e0797.png)